ionic-index.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
```

Comment:
New command for Ionic. Similar but different. I tried making this file simply invoke the original subsection generator with the `-t` option using `execFile` (from the module `child-process`). I got it to work but decided on this approach mostly because the errors were referring to the original subsection generator not the Ionic subsection generator. I thought that would be confusing (i.e. to run the command `yarn generate-ionic-subsection` and have errors mentioning `generate-subsection`).

Reply:
Since `index.js` is meant to indicate the main / default file, I don't think we need to add the `-index.js` suffix to this file

Reply:
👍